thinkflow/migrations/0001_initial.py

```
@@ -0,0 +1,29 @@
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='PlantDisease',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('uuid', models.UUIDField(editable=False, unique=True)),
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)